src/llm_vm/onsite_llm.py

```diff
@@ -527,3 +485,71 @@ def finetune(self, dataset, optimizer, c_id, small_model_filename=None):
         # optimizer.storage.set_training_in_progress(c_id, False)
         # if old_model is not None:
         #     openai.Model.delete(old_model)
+
+
+class BaseCtransformersLLM(BaseOnsiteLLM):
+    """
+    Base Class for running Ctransformers/GGML models
+
+    Attributes:
+        model_uri (str): Ctransformers uri for LLM
+        model_kwargs (dict): Keyword arguments for loading the LLM
+    
+    Methods:
+        model_loader: Loads specified model from Ctransformers
+        generate: Generates a response from given prompt
+
+    """
+
+    def __init__(self, **model_kwargs):
+        self.__model_uri = None
```

Comment:
Why the double underscores ?

Reply:
it means the attribute is only being used internally. similar to `__init__`